src/scss/includes/search_form.scss

```diff
@@ -254,6 +256,13 @@ input[type="search"] {
   padding: 9px 18px;
 }
 
+.autocomplete_error {
+  padding: 15px 10px 5px;
+  font-size: 16px;
```

Comment:
I don't think it's useful to redefine this `font-size`, as it's already `16px` on the parent element.